src/Generating/CompoGenBiomal.cpp

```diff
@@ -419,7 +419,7 @@ class cCompoGenBiomal :
 			top = m_SeaLevel;
 			a_ChunkDesc.SetHeight(a_RelX, a_RelZ, top - 1);
 		}
-		for (int y = top; y > 0; y--)
+		for (int y = top; y > cChunkDef::LowerLimit; y--)
```

Comment:
:exclamation:  This needs a rework, `a_ShapeColumn` is an array (0 .. cChunkDef::VerticalBlockCount), while `a_ChunkDesc::SetBlockType` requires chunk-relative Y coord (cChunkDef::LowerLimit .. cChunkDef::UpperLimit - 1)